### DISCUSSION TITLE
Can Pivpn (setup on a VPS server) somehow replace my need for Cloudflare Zero Trust with NPM?

### DISCUSSION THREAD

**Discussion by piptravels** [pivpn/pivpn#1706] [Support] [answered]
I am not in the tech field. I have recently created a homelab with Proxmox on an old laptop and a LXD server on an old Dell workstation where I run docker apps inside my lxc containers. I have gaping holes in my understanding but want something I've never had before and that is some semblance of privacy (with ad blocker either PiHole or Adguard Home)

I did all of this by studying and applying what I learned from Tech Youtube creators specifically Scottibyte and Awesome Open Source DB Tech and a few others. I can copy and paste as well as any other monkey.

I installed Pivpn on an Ubuntu VPS server and it installed correctly (as far as I know). Is there a GUI dashboard to set things up by chance? I saw that it's using port 51820 in the installation but when I went to my VPS ip address at https://198.91.27.x:51820 I got an error "connection refused" (and no I didn't have the 'x' in there :)

I have a GL-AR750S-EXT which has Openwrt installed (factory) and it has Wireguard settings in there but not sure yet how to connect the Router and VPS. Then I think all that would be left is Pihole or Adguard and Nginx Proxy Manager and I can rid myself of Cloudflare Zero trust (which is ironic because the more I learn about them the more my trust level of them drops to zero)

If someone can point me in the right direction on a tutorial on how to connect all of this I would be grateful. Thanks in advance.

**Comment by TinCanTech** [comment, marked as answer]
Yeah .. PiVPN documentation .. where is it ? [delete-as-necessary]
